front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -125,9 +127,70 @@ export function AgentMessage({
       pictureUrl={agentMessageToRender.configuration.pictureUrl}
       name={agentMessageToRender.configuration.name}
     >
-      <div className="text-base font-normal">
-        {agentMessageToRender.content}
-      </div>
+      {renderMessage(agentMessageToRender)}
     </ConversationMessage>
   );
 }
+
+function renderMessage(agentMessage: AgentMessageType) {
+  // Display the error to the user so they can report it to us (or some can be
+  // understandable directly to them)
+  if (agentMessage.status === "failed") {
+    return (
+      <div>
+        <div className="mb-2 text-base font-normal">
```

Comment:
As a first iteration let's just drop this one for now until we have a final design.